publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/config/RabbitMqPropertiesConfig.java

```diff
@@ -77,6 +80,8 @@
     private void readSpringProperties() {
         JsonNode rabbitmqInstancesJsonListJsonArray = null;
         final ObjectMapper objMapper = new ObjectMapper();
+        final String jasyptKey = readJasyptKeyFile(jasyptPassword);
```

Comment:
jasyptPassword is password or a jasypt key file path?

Reply:
jasyptPassword value is jasyptKey file path, so do I change variable to jasyptKeyFilePath?
I have question regarding property value of jasypt.encryptor.password in application.properties. Now we are passing the jasypt.encryptor.password value is jasypt key file path. So can I change this property name to jasypt.encryptor.jasyptKeyFilePath or something?

Reply:
yes please change. jasyptKeyFilePath is fine.

Reply:
Changed to jasyptKeyFilePath in application.properties and code.